backend/otel/otel_test.go

```diff
@@ -0,0 +1,34 @@
+package otel
+
+import (
+	"net/http"
+	"strings"
+	"testing"
+)
+
+type MockResponseWriter struct{}
+
+func (m *MockResponseWriter) Header() http.Header {
+	return http.Header{}
+}
+
+func (m *MockResponseWriter) Write(bytes []byte) (int, error) {
+	return 0, nil
+}
+
+func (m *MockResponseWriter) WriteHeader(statusCode int) {
+	return
+}
+
+func TestHandler_HandleLog(t *testing.T) {
+	w := &MockResponseWriter{}
+	r, _ := http.NewRequest("POST", "", strings.NewReader(""))
+	h := Handler{}
+	h.HandleLog(w, r)
```

Comment:
Nice. This will make it easier to trace through the code with various scenarios.